src/migrations.cr

```diff
@@ -2,7 +2,7 @@ require "./migrations/*"
 
 module Migrations
   def self.apply_all
-    {% for migration in Migration.includers.sort { |t| basename(filename t) } %}
+    {% for migration in Migration.includers.sort_by &.constant(:Ref) %}
```

Comment:
Note: original implementation was borked, but undetected due to the sort not being invoked. We should likely neaten this up, but this preserves the original intended behaviour. Open to suggestions on best approach.